ClientHints.php

```diff
@@ -222,13 +222,13 @@ public static function factory(array $headers): ClientHints
                 case 'sec-ch-ua-arch':
                 case 'arch':
                 case 'architecture':
-                    $architecture = $value;
+                    $architecture = \str_replace('"', '', $value);
```

Comment:
Just got another idea. We could simply use `trim` so any `"` in the middle would be untouched.
```suggestion
                    $architecture = \trim($value, '"');
```

Reply:
sounds good.
So it should be back at the beginning for all?

Reply:
No. That would still break `sec-ch-ua-full-version-list`, as that one requires the surrounding `"`.

Reply:
yes, I noticed that myself. :P